addons/website_mass_mailing/migrations/13.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Payam Yasaie <https://www.tashilgostar.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+_column_copies = {
+    'mailing_list': [
+        ('popup_content', None, None),
+    ],
+}
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.copy_columns(env.cr, _column_copies)
```

Comment:
I think you can do rename_columns instead.

Reply:
@Yasaie please have a look at this comment ^

Reply:
I don't remember why I said this. Maybe I was wrong here.

Reply:
This is not needed indeed.